client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,80 @@
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import YelpLogo from "../../../assets/images/yelp-logo.png";
+import RedirectButton from "../../../components/RedirectButton";
+
+const classes = {
+  previewContainer: {
+    height: "160px",
+    width: "100%",
+    backgroundColor: "gray",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "100%",
+    width: "100px",
+    marginLeft: "15px",
+    objectFit: "cover",
+  },
+  detailsContainer: {
+    marginLeft: "10px",
+  },
+  nameContainer: {
+    fontSize: "30px",
```

Comment:
might be too big. I think the largest font used on this website is `1.5rem`, not sure.